katas/helloworld/valeria/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Hello @svaleria , please remove `.vscode` folder and add it to `.gitignore` file, we will not version these configuration files yet